tests/product/test_configuration.py

```diff
@@ -337,16 +317,7 @@ def test_configuration_show_coord_worker_using_dash_x(self):
             expected = f.read()
         self.assertRegexpMatches(output, expected)
 
-    def test_configuration_no_presto_user(self):
```

Comment:
There is a test in `test_connectors` (`test_connector_add_no_presto_user`) that does the same thing but for adding a connector. I say we can get rid of this one and add a unit test that ensures that we try to add configuration files as the presto user.